app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -175,7 +175,7 @@ chmod 644 $KATELLO_SERVER_CA_CERT
 if ! [ -x "$(command -v subscription-manager)" ] ; then
   $PKG_MANAGER_INSTALL subscription-manager
```

Comment:
I reproduced the reported issue and tested it with the changes in this PR. For me, the current modifications cleared all the unnecessary messages.

Reply:
`$PKG_MANAGER_INSTALL subscription-manager` this one no, because if installation of subman fails, users need to know about it.

Reply:
OK, on a second thought if sub-man is not installed on the system, package manager should not complain about subscriptions, since the plugin that does that, would not be installed at all.